doc/user/content/serve-results/s3.md

```diff
@@ -0,0 +1,231 @@
+---
+title: "Amazon S3"
+description: "How to export results from Materialize to Amazon S3."
```

Comment:
Like MinIO, right? I was on the fence about mentioning it since we also don't explicitly mention it for Kafka or the Schema Registry. We just tend to spell out the compatible tools that are relevant to users and are likely to be used with Materialize. I can add a note, though!

Reply:
On the AWS Connection docs we do mention that this is compatible with AWS-compatible storage providers: https://materialize.com/docs/sql/create-connection/#aws-options

So I think we can be consistent with that